### PR TITLE
Hotfix demo

### PR DESCRIPTION
**Pull request by eineving**

### CODE REVIEW

**Comment by chraker** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [fixtures.sql#L62](https://github.com/SoftwareEvolutionProject/2017Group1/blob/720e3e85bc8650a3380c65fac3be1bb3276f580d/database/fixtures.sql#L62): Define a constant instead of duplicating this literal 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)
1. ![CRITICAL][CRITICAL] [fixtures.sql#L67](https://github.com/SoftwareEvolutionProject/2017Group1/blob/720e3e85bc8650a3380c65fac3be1bb3276f580d/database/fixtures.sql#L67): Define a constant instead of duplicating this literal 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)
1. ![CRITICAL][CRITICAL] [fixtures.sql#L72](https://github.com/SoftwareEvolutionProject/2017Group1/blob/720e3e85bc8650a3380c65fac3be1bb3276f580d/database/fixtures.sql#L72): Define a constant instead of duplicating this literal 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)
1. ![CRITICAL][CRITICAL] [fixtures.sql#L77](https://github.com/SoftwareEvolutionProject/2017Group1/blob/720e3e85bc8650a3380c65fac3be1bb3276f580d/database/fixtures.sql#L77): Define a constant instead of duplicating this literal 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)
1. ![CRITICAL][CRITICAL] [fixtures.sql#L87](https://github.com/SoftwareEvolutionProject/2017Group1/blob/720e3e85bc8650a3380c65fac3be1bb3276f580d/database/fixtures.sql#L87): Define a constant instead of duplicating this literal 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)
1. ![CRITICAL][CRITICAL] [fixtures.sql#L105](https://github.com/SoftwareEvolutionProject/2017Group1/blob/720e3e85bc8650a3380c65fac3be1bb3276f580d/database/fixtures.sql#L105): Define a constant instead of duplicating this literal 6 times. [rule](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)
1. ![CRITICAL][CRITICAL] [fixtures.sql#L110](https://github.com/SoftwareEvolutionProject/2017Group1/blob/720e3e85bc8650a3380c65fac3be1bb3276f580d/database/fixtures.sql#L110): Define a constant instead of duplicating this literal 6 times. [rule](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.html [hunk 1]
```diff
@@ -22,23 +22,61 @@ <h2 *ngIf="creating">Printing a Digital Print</h2>
...
+                            *ngFor="let _orderedPart of this.orderedParts; let i = index"
+                            [value]=_orderedPart.id>
+                      {{_orderedPart.id}}
+                  </select>
+                </div>
+              </div>
+              <br>
+            </div>
+
+            <!--&lt;!&ndash;<app-digital-print-details-panel *ngIf="digitalPrint" [digitalPrint]="digitalPrint"-->
```
Comment (chraker) [inline frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.html:66]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

File: frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.html [hunk 2]
```diff
@@ -22,23 +22,61 @@ <h2 *ngIf="creating">Printing a Digital Print</h2>
...
+                                             <!--[digitalParts]="digitalParts" [close]="false">-->
```
Comment (chraker) [inline frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.html:67]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

File: frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.html [hunk 3]
```diff
@@ -22,23 +22,61 @@ <h2 *ngIf="creating">Printing a Digital Print</h2>
...
+            <!--</app-digital-print-details-panel>&ndash;&gt;-->
```
Comment (chraker) [inline frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.html:68]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

**Comment by chraker** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by mlonn — APPROVED** [review]

**Pull request merged**